src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -1,67 +1,66 @@
-  --#{$text-input-group}__text--before--BorderColor: var(--#{$pf-global}--BorderColor--300);
-
-  // Bottom border
-  --#{$text-input-group}__text--after--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$text-input-group}__text--after--BorderBottomColor: var(--#{$pf-global}--BorderColor--200);
-  --#{$text-input-group}--hover__text--after--BorderBottomColor: var(--#{$pf-global}--primary-color--100);
-  --#{$text-input-group}__text--focus-within--after--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--md);
-  --#{$text-input-group}__text--focus-within--after--BorderBottomColor: var(--#{$pf-global}--primary-color--100);
+  // Border hover
+  --#{$text-input-group}--m-hover--BorderColor: var(--pf-t--global--border--color--hover);
 
   // Main
-  --#{$text-input-group}__main--first-child--not--text-input--MarginLeft: var(--#{$pf-global}--spacer--sm);
-  --#{$text-input-group}__main--m-icon__text-input--PaddingLeft: var(--#{$pf-global}--spacer--xl);
-  --#{$text-input-group}__main--RowGap: var(--#{$pf-global}--spacer--xs);
-  --#{$text-input-group}__main--ColumnGap: var(--#{$pf-global}--spacer--sm);
+  --#{$text-input-group}__main--first-child--not--text-input--MarginLeft: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}__main--m-icon__text-input--PaddingLeft: var(--pf-t--global--spacer--xl);
+  --#{$text-input-group}__main--RowGap: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}__main--ColumnGap: var(--pf-t--global--spacer--xs);
 
   // Chip group
-  --#{$text-input-group}--c-chip-group__main--PaddingTop: var(--#{$pf-global}--spacer--xs);
-  --#{$text-input-group}--c-chip-group__main--PaddingRight: var(--#{$pf-global}--spacer--xs);
-  --#{$text-input-group}--c-chip-group__main--PaddingBottom: var(--#{$pf-global}--spacer--xs);
+  --#{$text-input-group}--c-chip-group__main--PaddingTop: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}--c-chip-group__main--PaddingRight: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}--c-chip-group__main--PaddingBottom: var(--pf-t--global--spacer--xs);
 
   // Text input
   --#{$text-input-group}__text-input--PaddingTop: var(--#{$pf-global}--spacer--form-element);
-  --#{$text-input-group}__text-input--PaddingRight: var(--#{$pf-global}--spacer--sm);
+  --#{$text-input-group}__text-input--PaddingRight: var(--pf-t--global--spacer--sm);
   --#{$text-input-group}__text-input--PaddingBottom: var(--#{$pf-global}--spacer--form-element);
-  --#{$text-input-group}__text-input--PaddingLeft: var(--#{$pf-global}--spacer--sm);
+  --#{$text-input-group}__text-input--PaddingLeft: var(--pf-t--global--spacer--sm);
   --#{$text-input-group}__text-input--MinWidth: 12ch;
-  --#{$text-input-group}__text-input--m-hint--Color: var(--#{$pf-global}--Color--200); // matches placeholder text color
-  --#{$text-input-group}__text-input--placeholder--Color: var(--#{$pf-global}--Color--200);
+  --#{$text-input-group}__text-input--m-hint--Color: var(--pf-t--global--text--color--default);
```

Comment:
Just want to confirm we don't want this to be a grey color, or different from default text? It's the "autocomplete" text that shows up in the input as you're typing if your text matches an item in a menu or whatever that the input can autocomplete.